<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mdtSqlCopierCodecSettingsWidget</class>
 <widget class="QWidget" name="mdtSqlCopierCodecSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Codec settings</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lbType">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="cbType"/>
     </item>
     <item>
      <widget class="QToolButton" name="tbSelect">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbConnectionNameLabel">
     <property name="text">
      <string>Connection:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lbConnection">
     <property name="text">
      <string>cnn</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbFileLabel">
     <property name="text">
      <string>File:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lbFile">
     <property name="text">
      <string>file</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lbDirectoryLabel">
     <property name="text">
      <string>Directory:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="lbDirectory">
     <property name="text">
      <string>directory</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lbTable">
     <property name="text">
      <string>Table:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="cbTable"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
